src/client/scripts/game/chess/checkdetection.js

```diff
@@ -314,6 +314,18 @@ const checkdetection = (function(){
         return true;
     }
 
+    /**
+     * 
+     * @param {gamefile} gamefile 
+     * @param {String} color 
+     * @returns {Boolean}
+     */
+    function areWeInCheck (gamefile, color) {
```

Comment:
Can you finish the jsdoc here? I had to study it's logic to be able to tell what it does. My first thought was *Is it the same as `checkdetection.detectCheck()`?*